script/Deploy_lending_optimism.s.sol

```diff
@@ -0,0 +1,93 @@
+/**
+ * Created by Arcadia Finance
+ * https://www.arcadia.finance
+ *
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity ^0.8.13;
+
+import "../lib/forge-std/src/Test.sol";
+import { DeployAddresses, DeployNumbers, DeployBytes, DeployRiskConstants } from "./Constants/DeployConstants.sol";
+
+import { Factory } from "../src/Factory.sol";
+import { Liquidator } from "../src/Liquidator.sol";
+
+import { DataTypes } from "../lib/arcadia-lending/src/libraries/DataTypes.sol";
+import { ERC20, DebtToken } from "../lib/arcadia-lending/src/DebtToken.sol";
+import { LendingPool } from "../lib/arcadia-lending/src/LendingPool.sol";
+import { Tranche } from "../lib/arcadia-lending/src/Tranche.sol";
+import { TrustedCreditor } from "../lib/arcadia-lending/src/TrustedCreditor.sol";
+
+contract ArcadiaLendingDeployerOptimism is Test {
+    Factory public factory;
+    ERC20 public weth;
+    ERC20 public usdc;
+    Liquidator public liquidator;
+
+    LendingPool public pool_weth;
+    Tranche public srTranche_weth;
+    Tranche public jrTranche_weth;
+
+    LendingPool public pool_usdc;
+    Tranche public srTranche_usdc;
+    Tranche public jrTranche_usdc;
+
+    constructor() {
+        weth = ERC20(DeployAddresses.eth_optimism);
+        usdc = ERC20(DeployAddresses.usdc_optimism);
+    }
+
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY_DEPLOYER");
+
+        vm.startBroadcast(deployerPrivateKey);
+
+        factory = new Factory();
+        liquidator = new Liquidator(address(factory));
+        liquidator.setAuctionCurveParameters(3600, 14_400);
+
+        pool_weth =
+            new LendingPool(ERC20(address(weth)), DeployAddresses.treasury, address(factory), address(liquidator));
+        srTranche_weth = new Tranche(address(pool_weth), "Senior", "s");
+        jrTranche_weth = new Tranche(address(pool_weth), "Junior", "j");
+
+        pool_weth.setOriginationFee(10);
+        pool_weth.setMaxInitiatorFee(66 * 10 ** 18);
```

Comment:
Should be much much lower on optimism?
set, fixedLiquidationCost also, lower as on mainnet

Reply:
3 eth / 5k usdc on optimism, 33 eth / 50k on mainnet